bulletin-de-la-societe-entomologique-de-france.csl

```diff
@@ -0,0 +1,170 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="fr-FR" xmlns="http://purl.org/net/xbiblio/csl">
+  <info>
+    <title>Bulletin de la Société Entomologique de France</title>
+    <title-short>bsef</title-short>
+    <id>http://www.zotero.org/styles/bulletin-de-la-societe-entomologique-de-france</id>
+    <link href="http://www.zotero.org/styles/bulletin-de-la-societe-entomologique-de-france" rel="self"/>
+    <link href="http://www.zotero.org/styles/bmj" rel="template"/>
+    <link href="http://www.lasef.org/publications/le-bulletin-de-la-sef/" rel="documentation"/>
```

Comment:
@KwAntynn, this link doesn't work for me. I'll link to http://www.lasef.org/new/new_bulletin.html instead.